gammapy/utils/regions.py

```diff
@@ -137,6 +144,44 @@ def regions_to_compound_region(regions):
     return region_union
 
 
+def get_centroid(vertices):
+    """Compute centroid of a polygon. Implicitly assumes a flat
+    cartesian projection, will probably break for very large polygons.
+
+    Code comes from:
+    https://stackoverflow.com/questions/75699024/finding-the-centroid-of-a-polygon-in-python
```

Comment:
@ParsonsRD it's fine to acknowledge it like this, we have the same in other places in gammapy